packages/liveblocks-client/src/immutable.test.ts

```diff
@@ -172,12 +158,9 @@ describe("2 ways tests with two clients", () => {
 
   describe("Array/LiveList", () => {
     test("add item to array", async () => {
-      const { storage, state, assert } = await prepareStorageImmutableTest<
-        {
-          syncList: LiveList<string>;
-        },
-        { syncList: string[] }
-      >(
+      const { storage, state, assert } = await prepareStorageImmutableTest<{
+        syncList: LiveList<string>;
+      }>(
```

Comment:
The (JSON) state now gets automatically derived from the provided (LSON) storage type. This means, further down on L172, `state.syncList` will get inferred as `string[]` automatically, from the provided `LiveList<string>` type here.